### DISCUSSION TITLE
[Help]: Dampening / Mass options for slide control

### DISCUSSION THREAD

**Discussion by tayteboss** [davidjerleke/embla-carousel#826] [Help]
### Summary

This could be a feature request. However, I was wondering if there are any options or ways to control the mass, bounce or dampening as you scroll through slides. You see this in libraries such as Framer Motion which also use a physics-based approach to motion. I love this carousel, however, sometimes I find when you scroll it feels too 'light' when it could be great to have a 'heavier' feeling when scrolling through slides.

Thanks!

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@tayteboss Thank you for the suggestion! Exposing physics options like mass, bounce, or dampening is unfortunately a lot of work. The reason is that these settings can easily break the carousel animations, which would lead to a large number of bug reports and a huge maintenance burden. I simply don’t have the spare time to manage that right now.

If this project were to receive decent funding, I would have the bandwidth to look into safely exposing physics configurations. Until then, it’s not something I can support.
